engine/table/build.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
     // TODO(deephaven-core#3204): t-digest 3.3 appears to have higher errors than 3.2
     implementation 'com.tdunning:t-digest:3.2'
     implementation 'com.squareup:javapoet:1.13.0'
+    implementation 'io.github.classgraph:classgraph:4.8.103'
```

Comment:
I like classgraph. We need to decide if this additional dependency is worth it. It doesn't have any external dependencies itself, which is nice.

I'd bump to the latest version.

Reply:
I've bumped. I'd be interested in any other approach you have in mind. The reported bug is an annoying thing for the user to need to handle.

Maybe we should make groovy-support an extension?